api/MediaDevices.json

```diff
@@ -499,6 +499,69 @@
             "deprecated": false
           }
         }
+      },
+      "selectAudioOutput": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/selectAudioOutput",
+          "spec_url": "https://w3c.github.io/mediacapture-output/#dom-mediadevices-selectaudiooutput",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "88",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "media.setsinkid.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": "88",
```

Comment:
Ought to be `false`, unless https://github.com/mdn/browser-compat-data/issues/9679 has changed.

Reply:
Yes indeed. How quickly I forget :-(. Fixed.